pubsub/jetstream/jetstream.go

```diff
@@ -255,7 +253,22 @@ func (js *jetstreamPubSub) Subscribe(ctx context.Context, req pubsub.SubscribeRe
 		subscription, err = js.jsc.QueueSubscribe(req.Topic, queue, natsHandler, nats.Bind(streamName, consumerInfo.Name))
 	} else {
 		js.l.Debugf("nats: subscribed to subject %s", req.Topic)
-		subscription, err = js.jsc.Subscribe(req.Topic, natsHandler, nats.Bind(streamName, consumerInfo.Name))
+		subscription, err = js.jsc.Subscribe(
+			req.Topic,
+			func(msg *nats.Msg) {
```

Comment:
Nit: no need to capture the parameter in the closure since it's not in a loop

```suggestion
			func() {
```